lib/utils/github.test.ts

```diff
@@ -9,6 +10,18 @@ import {
 } from "lib/utils/github";
 
 describe("[lib] github methods", () => {
+  beforeEach(() => {
+    vi.mock("@supabase/auth-helpers-nextjs", async () => {
```

Comment:
Mocking `@supabase/auth-helpers-nextjs` so that these tests can run. It wasn't doing much before with this when using jest, so this is a good enough mock.